CMakeLists.txt

```diff
@@ -6,14 +6,14 @@ if(NOT CMAKE_BUILD_TYPE)
   set(CMAKE_BUILD_TYPE "Release" CACHE STRING "Choose the type of build." FORCE)
 endif()
 
-project(vowpal_wabbit C CXX)
-set(CMAKE_CXX_STANDARD 11)
-
 # VW targets Windows 8.1 SDK
 if(WIN32)
   set(CMAKE_SYSTEM_VERSION 8.1 CACHE TYPE INTERNAL FORCE)
 endif()
 
+project(vowpal_wabbit C CXX)
```

Comment:
[Minor] Is there a reason this line needs to be below the Win8.1 SDK targetting setting? If so, consider adding a comment about this.